### PR TITLE
[BD-46] feat: removed white background on Stepper.Header component

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

Issue: https://github.com/openedx/paragon/issues/2342

### Deploy Preview

[Stepper component](https://deploy-preview-2358--paragon-openedx.netlify.app/components/stepper/)

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [x] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [x] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [x] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [x] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

**Review by bydawen — APPROVED** [review]

File: src/Stepper/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+$stepper-header-step-border:                              none !default;
+$stepper-header-step-min-width:                           0 !default;
+$stepper-header-step-padding:                             .25rem !default;
+$stepper-header-step-list-padding-y:                      .25rem !default;
+$stepper-header-step-list-padding-x:                      0 !default;
+$stepper-header-step-list-margin:                         0 !default;
+$stepper-header-active-step-bg:                           $gray-500 !default;
+$stepper-header-step-error-bubble-color:                  $danger !default;
+$stepper-header-step-error-bubble-box-shadow-color:       $stepper-header-step-error-bubble-color !default;
+$stepper-header-step-error-bubble-box-shadow-width:       3px !default;
```
Comment (monteri) [inline src/Stepper/_variables.scss:18]: I would reduce the name to 5 words max. In this case readability of variable is complicated, although the idea as I understand is to make it more clear what this value means.

Comment (PKulkoRaccoonGang) [reply src/Stepper/_variables.scss]: It`s interesting to think. Today we don't have special rules for naming SCSS variables. I agree with your idea about the unreadability of long names in SCSS variables. Unfortunately, in this case, I can't come up with a good combination for this variable (using 5 words). Would you happen to have any suggestions for this?

File: src/Stepper/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+$stepper-header-step-bg:                                  $stepper-header-bg !default;
+$stepper-header-step-border:                              none !default;
+$stepper-header-step-min-width:                           0 !default;
+$stepper-header-step-padding:                             .25rem !default;
+$stepper-header-step-list-padding-y:                      .25rem !default;
+$stepper-header-step-list-padding-x:                      0 !default;
+$stepper-header-step-list-margin:                         0 !default;
+$stepper-header-active-step-bg:                           $gray-500 !default;
+$stepper-header-step-error-bubble-color:                  $danger !default;
+$stepper-header-step-error-bubble-box-shadow-color:       $stepper-header-step-error-bubble-color !default;
```
[the remark this answers is not in the document]
Comment (PKulkoRaccoonGang) [reply src/Stepper/_variables.scss]: Thanks

**Review by adamstankiewicz — APPROVED** [review]
Nice refactor with the addition of the `_variables.scss` file :)  

[question] Curious, will we need to create or do we already have these stepper SCSS variables as design tokens / CSS variables?

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/2358&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2023-06-06+13:02&entry.1917517419=2023-06-08+11:57&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by PKulkoRaccoonGang** [comment]
> Nice refactor with the addition of the `_variables.scss` file :)
> 
> [question] Curious, will we need to create or do we already have these stepper SCSS variables as design tokens / CSS variables?

Thanks 👍 
It will be necessary to create

**Pull request merged**